config/default.yml

```diff
@@ -96,6 +96,16 @@ FactoryBot/FactoryNameStyle:
   VersionChanged: '2.23'
   Reference: https://www.rubydoc.info/gems/rubocop-factory_bot/RuboCop/Cop/FactoryBot/FactoryNameStyle
 
+FactoryBot/IdSequence:
+  Description: Do not create a FactoryBot sequence for an id column.
+  Enabled: pending
+  Include:
```

Comment:
Is this needed/complete?
I see we’re about to extend our base list of patterns, and add a prefix wildcard to patterns, too.

Reply:
@owst The default Include path has been changed by https://github.com/rubocop/rubocop-factory_bot/pull/55 . Could you please check?

Reply:
Thanks for the ping @ydah - done!